WordPress/src/main/res/values/strings.xml

```diff
@@ -977,6 +977,7 @@
     <string name="editor_post_settings_featured_image">Featured Image</string>
     <string name="editor_post_settings_set_featured_image">Set Featured Image</string>
     <string name="post_settings_status">Status</string>
+    <string name="post_settings_format">Post Format</string>
```

Comment:
Fixed in a8ef3a61e2b708223288500d4638529fba718852. I blame AS for not graying out the unused resource 🤒